common/cluster-api/src/main/proto/queue.proto

```diff
@@ -1012,6 +1019,7 @@ message ToDeviceActorNotificationMsgProto {
   ToDeviceRpcRequestActorMsgProto toDeviceRpcRequestMsg = 5;
   FromDeviceRpcResponseActorMsgProto fromDeviceRpcResponseMsg = 6;
   RemoveRpcActorMsgProto removeRpcActorMsg = 7;
+  DeviceDeletedMsgProto deviceAssignToTenantMsgProto = 8;
```

Comment:
Is it `deviceDeletedMsg` or `deviceAssignedMsg`? All variables here end with Msg and not MsgProto.